test/release/examples/benchmarks/shootout/chameneosredux.chpl

```diff
@@ -74,6 +74,12 @@ proc simulate(numChameneos) {
   // print information about the population once we're done
   //
   printInfo(chameneos);
+
+  //
+  // hasta las chameneos, baby.
+  //
+  destroyChameneos(chameneos);
+  delete meetingPlace;
```

Comment:
I can make the chameneos change. If the iterator change doesn't make a difference outside of your branch, we might as well leave it there :)